poly-synth/src/poly-synth.js

```diff
@@ -32,15 +32,38 @@ class PolySynth {
     this.releasedVoices_ = [];
 
     // The maximum and minimum cutoffs are experimentally determined.
-    this.lowPassMaxCutoff = 16000;
-    this.lowPassMinCutoff = 60;
+    this.maxCutoff = 16000;
```

Comment:
Be that as it may, churning names in the middle of a review when no reviewer asked for such a change just makes the review take longer since there are now more things to review that didn't need to be reviewed.